test/powershell/Language/Parser/Ast.Tests.ps1

```diff
@@ -25,8 +25,8 @@ Describe "The SafeGetValue method on AST returns safe values" -Tags "CI" {
             throw "No Exception!"
         }
         catch {
-            $_.FullyQualifiedErrorId | Should be "InvalidOperationException"
-            $_.ToString() | Should Match '\$a'
+            $_.FullyQualifiedErrorId | Should -BeExactly "InvalidOperationException"
```

Comment:
In general, since we're making changes in these, we should use the new syntax for pester when it checks for throw. The `try/catch` we used then we don't need anymore.
```
{ $ast.SafeGetValue() } |Should -Throw -ErrorId "InvalidOperationException
```